.chloggen/new-service-instance-id.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: Operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added the service.instance.id as os.Hostname into the traces resource Env.
```

Comment:
Is this accurate? I don't see how the hostname is used for the service ID


> Added the service.instance.id as os.Hostname into the resource attributes.

Reply:
It was not accurate since I changed to UUID in the meantime. Anyway, I've put the pod.UID to be the service.instance.id.